pkg/controller/siddhiprocess/operator.go

```diff
@@ -175,7 +175,7 @@ func (rsp *ReconcileSiddhiProcess) deployApp(
 func (rsp *ReconcileSiddhiProcess) deployParser(sp *siddhiv1alpha2.SiddhiProcess, configs Configs) (err error) {
 	reqLogger := log.WithValues("Request.Namespace", sp.Namespace, "Request.Name", sp.Name)
 	siddhiApp := SiddhiApp{
-		Name: sp.Name,
+		Name: sp.Name + ParserExtension,
```

Comment:
The `sp.Name + ParserExtension` is duplicated multiple times. You can use a variable to hold this value.

Reply:
Fixed in https://github.com/siddhi-io/siddhi-operator/pull/123/commits/c7610e754ce20646bd6d415b1c650ce127064af3